charts/edge/README.md

```diff
@@ -14,11 +14,15 @@
 ```sh
 helm install edge mezmo/edge \
   --namespace <MY_NAMESPACE> \
-  --set mezmoApiAccessToken=<MEZMO_API_ACCESS_TOKEN> \
-  --set service.sourcePorts.start=<START_PORT> \
-  --set service.sourcePorts.end=<END_PORT> \
+  --set mezmoApiAccessToken=<MEZMO_API_ACCESS_TOKEN>
 ```
 
+By default ports in the range [8000, 8010] are configured. See `service.sourcePorts.*` values for configuring different ranges. For example, to configure with no contiguous range and two ports (sysloc and http) exposed:
+
+```sh
+helm install edge mezmo/edge \
+  --set mezmoApiAccessToken=<MEZMO_API_ACCESS_TOKEN>,service.sourcePorts.end=0,service.sourcePorts.list="{514,80}
```

Comment:
Missing line breaks and `--set` per parameter.

Reply:
are you saying you want those separated as a matter of style? Because this conforms to helm command usage (except it looks like I am missing a quote at the end here, which I will fix).

Reply:
TIL that it works
I think for readability and to avoid horizontal scroll in GitHub / other UIs.

Reply:
I pushed it with split lines. Now looking into a test failure I was not expecting.